modules/ting_search_carousel/ting_search_carousel.install

```diff
@@ -19,3 +20,37 @@ function ting_search_carousel_uninstall() {
   variable_del('ting_carousel_search_queries_hidden');
   variable_del('ting_carousel_search_queries');
 }
+
+function ting_search_carousel_add_shortcut() {
+  // Add shortcut to carousel.
+  $shortcut_set = shortcut_set_load(SHORTCUT_DEFAULT_SET_NAME);
+  $shortcut_set->links = array(
+    array(
+      'link_path' => 'admin/config/ting/ting_search_carousel',
+      'link_title' => st('Configure carousel'),
+      'weight' => -14,
+    ),
+  );
```

Comment:
Will this in effect reset the shortcut set to only contain this shortcut? How about adding it to the `links` array instead?